web-client/src/components/DashboardLayout/DashboardLayout.tsx

```diff
@@ -2,9 +2,9 @@ import { Layout } from 'antd';
 import React, { useEffect, useState } from 'react';
```

Comment:
Changed offers to specificOffers.  offers directory is for offers which are responses to requests. In new terminology, these are called specific offers.